config/locales/en.yml

```diff
@@ -7,4 +7,5 @@ en:
         invalid_content_type: "You are not allowed to upload %{content} file format. Allowed types: %{permitted}."
         invalid_extension: "You are not allowed to upload %{extension} file extension. Allowed types: %{permitted}."
   refile:
-    empty_param: "an empty"
\ No newline at end of file
+    empty_param: "an empty"
+    wrong_association_method: "wrong association name %{method}, use like this %{sugest}"
```

Comment:
Not necessary to use i18n here, it is a programmer error, just use a string directly on lib/refile/attachment/active_record.rb.